docs/docs/analytics.md

```diff
@@ -0,0 +1,17 @@
+# Optional Data Collection
+
+If you choose to allow Mathesar to collect analytics data, the following information is sent every 24 hours:
```

Comment:
```suggestion
If you allow Mathesar to collect analytics data, the following information is sent every 24 hours:
```

Small rephrasing suggestion. Also, I can see this sentence prompting the question "sent where?" Perhaps there's an easy, not too technical way to explain where we're storing the information but I'm not sure.

These are both nitpicks that can be ignored.